examples/resource_lacework_report_rule/main.tf

```diff
@@ -0,0 +1,101 @@
+terraform {
+  required_providers {
+    lacework = {
+      source = "lacework/lacework"
+    }
+  }
+}
+
+resource "lacework_report_rule" "example" {
+  name             = var.name
+  description      = var.description
+  enabled         = true
+  severities      = var.severities
+  resource_groups = var.resource_groups
+  email_alert_channels = var.channels
+
+  aws_compliance_reports {
+    pci = var.aws_pci
+    cis_s3 = true
+  }
+
+  gcp_compliance_reports {
+    pci = var.gcp_pci
+    cis = true
+  }
+
+  daily_compliance_reports {
+    aws_cloudtrail = var.daily_cloudtrail
+  }
+
+  weekly_snapshot = var.snapshot
+}
+
+variable "name" {
+  type    = string
+  default = "Report Rule 1"
+}
+
+variable "description" {
+  type    = string
+  default = "Report Rule created by Terraform"
+}
+
+variable "severities" {
+  type    = list(string)
+  default = ["High", "Medium"]
+}
+
+variable "resource_groups" {
+  type    = list(string)
+  default = ["TECHALLY_8416B4ADCED28565254842AA5906B729174653E1725F107"]
```

Comment:
what happens if we accidentally remove this group, or how do we set this up for QA environments?

Reply:
I will add in the tf inside the example to create and destroy this. Similar to what we did for alert rules